src/Spec2-Core/SpPresenter.class.st

```diff
@@ -115,6 +115,16 @@ SpPresenter class >> iconNamed: aSymbol [
 	^ Smalltalk ui icons iconNamed: aSymbol
 ]
 
+{ #category : #'labelled-presenters' }
+SpPresenter class >> iconWidth [
+	^ 24
+]
+
+{ #category : #'labelled-presenters' }
+SpPresenter class >> labelWidth [
```

Comment:
These two methods belonged to Spec-Polywidgets as extensions, but they show a super ugly dependency between the examples and the main presenters. Presenters should not understand `#labelWidth` and `#iconWidth`.

Reply:
I agree. 
And there are a lot other that we need to remove too (all related to direct manipulation of style and layouts).